x-pack/legacy/plugins/apm/public/components/app/ServiceMap/index.tsx

```diff
@@ -28,45 +25,14 @@ interface ServiceMapProps {
   serviceName?: string;
 }
 
-const cytoscapeDivStyle = {
-  background: `linear-gradient(
-  90deg,
-  ${theme.euiPageBackgroundColor}
-    calc(${theme.euiSizeL} - calc(${theme.euiSizeXS} / 2)),
-  transparent 1%
-)
-center,
-linear-gradient(
-  ${theme.euiPageBackgroundColor}
-    calc(${theme.euiSizeL} - calc(${theme.euiSizeXS} / 2)),
-  transparent 1%
-)
-center,
-${theme.euiColorLightShade}`,
-  backgroundSize: `${theme.euiSizeL} ${theme.euiSizeL}`,
-  margin: `-${theme.gutterTypes.gutterLarge}`,
-  marginTop: 0
-};
-const BetaBadgeContainer = styled.div`
-  right: ${theme.gutterTypes.gutterMedium};
-  position: absolute;
-  top: ${theme.gutterTypes.gutterSmall};
-  z-index: 1; /* The element containing the cytoscape canvas has z-index = 0. */
-`;
-
 export function ServiceMap({ serviceName }: ServiceMapProps) {
   const license = useLicense();
-  const { search } = useLocation();
-  const { urlParams, uiFilters } = useUrlParams();
+  const { urlParams } = useUrlParams();
   const params = useDeepObjectIdentity({
     start: urlParams.start,
     end: urlParams.end,
     environment: urlParams.environment,
-    serviceName,
-    uiFilters: {
-      ...uiFilters,
-      environment: undefined
-    }
+    serviceName
   });
```

Comment:
Redeclaring `urlParams` (and using `useDeepObjectIdentity`) is not necessary. We can simply pass `urlParams` to `useFetcher`